SpaceInvaders/Aliens/Strategies/ShootRandomlyStrategy.cs

```diff
@@ -7,8 +7,6 @@ namespace SpaceInvaders.Aliens.Strategies
 {
     public class ShootRandomlyStrategy : IShootStrategy
     {
-        private readonly Random _rng = new Random();
```

Comment:
@leppie, 
Making this "static" would only fix the Aliens mirroring behavior.
It will not fix the Strategy selector. Which currently selects the same shooting strategy for both players.
Refactoring to a single StaticRandom class solves both the mirroring and strategy selection. It also ensures both static Random classes do not have the same seed value.